ls_1/index.html

```diff
@@ -0,0 +1,1230 @@
+                    inventore veritatis et quasi architecto beatae vitae dicta
+                    sunt explicabo.
+                  </p>
+                </div>
+                <div class="start__buttons buttons-block">
+                  <button class="btn__blue_apply" type="button">
+                    Apply Now
+                  </button>
+                  <button class="btn__blue_apply" type="button">
+                    Request Service
+                  </button>
+                </div>
+              </div>
+            </div>
+            <div class="start__links">
+              <div class="start__links-title">
+                <img
+                  class="start__links-title-img"
+                  width="24"
+                  src="./images/white-chain.svg"
+                  alt="Chain logo"
+                />
+                <h2 class="start__links-title-text">Quick Link</h2>
+              </div>
+              <ul class="start__links-list">
+                <li class="start__item">
+                  <img
+                    class="start__item-img"
+                    width="24"
+                    src="./images/white-pen.svg"
+                    alt="Pen logo"
+                  /><a class="start__item-link" href="">Tution and Fees</a>
+                </li>
+                <li class="start__item">
+                  <img
+                    class="start__item-img"
+                    width="24"
+                    src="./images/white-briefcase.svg"
+                    alt="Briefcase logo"
+                  /><a class="start__item-link" href="">Review & Rating</a>
+                </li>
+                <li class="start__item">
+                  <img
+                    class="start__item-img"
+                    width="24"
+                    src="./images/white-book-open.svg"
+                    alt="Book logo"
+                  /><a class="start__item-link" href=""
+                    >Universoty Facilities</a
+                  >
+                </li>
+                <li class="start__item">
+                  <img
+                    class="start__item-img"
+                    width="24"
+                    src="./images/white-comment.svg"
+                    alt="Comment logo"
+                  /><a class="start__item-link" href="">Community Q&A</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </section>
+        <!-- Blog -->
+        <section class="blog" id="blog">
+          <div class="blog__container">
+            <div class="blog__title title">
+              <h2 class="title__text">Which one do you prefer?</h2>
+              <div class="title__border">
+                <img
+                  class="title__border-img"
+                  width="60"
+                  src="./images/blue-border.png"
+                  alt="border"
+                />
+              </div>
```

Comment:
You reuse this element more than one time. You can try to create pseudo element 'after' with this content and reuse it. Also one element in wrapper. And one more, it's just a rectangle maybe better create it using html+css and don't use image.